.claude/skills/watch-example-errors/SKILL.md

```
@@ -0,0 +1,86 @@
+scripts/example-error-watch.sh --platforms ios # faster
+scripts/example-error-watch.sh --build         # rebuild SDK dist first
+```
+Exit `0` clean · `1` errors · `2` setup fail. Catches bundle-time errors only
+(unresolved modules, syntax, transform). Does not execute JS.
+
+## Mode 2 — live background agent (warn me while I work)
+
+This is the "wrap a background shell with an agent that re-enters the session"
+setup. Two background shells + the agent (you):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)